Tests/Validator/UniqueUrlValidatorTest.php

```diff
@@ -111,10 +111,6 @@ public function testValidateWithPageDynamic()
 
     private function getContext()
     {
-        return $this->createMock(
-            class_exists('Symfony\Component\Validator\Context\ExecutionContextInterface') ?
-            'Symfony\Component\Validator\Context\ExecutionContextInterface' :
-            'Symfony\Component\Validator\ExecutionContextInterface'
-        );
+        return $this->createMock('Symfony\Component\Validator\Context\ExecutionContextInterface');
```

Comment:
`ExecutionContextInterface::class`

Reply:
@Soullivaneuh isn't there a fixer available which we could activate?

Reply:
Fixed! And tests pass now